obj/createTask.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    //DOM elements
+    submitButton : '[type="submit"]',
+    taskName : '[id="taskDetailsForm.name"]',
+    taskDescription : '[id="taskDetailsForm.description"]',
+    taskBill : '.Select-value',
+    taskBillSelect : '[aria-label="?"]',
+    taskBillSelected : '[aria-selected="true"]',
+    taskRate : '[id="taskDetailsForm.rate"]',
```

Comment:
If you selecting by id only you should use # instead, tricky part is to "escape dot" in order for it to be treated not as a class. so use this locator "#taskDetailsForm\.rate" same for rest of id fields
https://stackoverflow.com/questions/12310090/css-selector-with-period-in-id